src/core/server/metrics/metrics_service.ts

```diff
@@ -54,6 +60,10 @@ export class MetricsService
 
     const metricsObservable = this.metrics$.asObservable();
 
+    this.setupOpsMetricsEventType(analytics, metricsObservable);
+
+    metricsObservable.subscribe();
```

Comment:
I don't think this subscription is necessary, given you also subscribe in `setupOpsMetricsEventType`?